Week-2/Assignment 2.1/index.html

```diff
@@ -1,16 +1,61 @@
-    <h1>Heading 2</h1>
-    <article>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nobis velit aut laborum, tenetur repellendus natus. Fugit ab possimus sit tempore aspernatur, autem itaque debitis aliquam iure aliquid voluptates, repudiandae quis?</article>
-    <h1>Heading 3</h1>
-    <article>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nobis velit aut laborum, tenetur repellendus natus. Fugit ab possimus sit tempore aspernatur, autem itaque debitis aliquam iure aliquid voluptates, repudiandae quis?</article>
-    
+    <div class="bg">
+    </div>
+   
+    <header>
+        <h1>Welcome to the Matrix!</h1><br> 
+        - Wanna See how far the rabit hole goes?
+    </header>
+    <div class="content">
+    <article>
+        <h2>The Red Pill</h2>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Provident, ratione voluptates. Itaque aperiam ea ex harum voluptatibus blanditiis architecto expedita minus, molestiae earum enim modi dolore porro! Deserunt, saepe rem.</p>    
+        <a href="#">Link 1</a>
+    </article>
+    <article>
+        <h2>The Blue Pill</h2>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Provident, ratione voluptates. Itaque aperiam ea ex harum voluptatibus blanditiis architecto expedita minus, molestiae earum enim modi dolore porro! Deserunt, saepe rem.</p>    
+        <a href="#">Link 2</a>
+    </article>
+    <article>
+        <h2>The Zion</h2>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Provident, ratione voluptates. Itaque aperiam ea ex harum voluptatibus blanditiis architecto expedita minus, molestiae earum enim modi dolore porro! Deserunt, saepe rem.</p>    
+        <a href="#">Link 3</a>
+    </article>
+    <article>
+        <h2>Morbeous</h2>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Provident, ratione voluptates. Itaque aperiam ea ex harum voluptatibus blanditiis architecto expedita minus, molestiae earum enim modi dolore porro! Deserunt, saepe rem.</p>    
+        <a href="#">Link 4</a>
+    </article>
+    <article>
+        <h2>The Agents</h2>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Provident, ratione voluptates. Itaque aperiam ea ex harum voluptatibus blanditiis architecto expedita minus, molestiae earum enim modi dolore porro! Deserunt, saepe rem.</p>    
+        <a href="#">Link 5</a>
+    </article>
+    <article>
+        <h2>The Phonebooth</h2>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Provident, ratione voluptates. Itaque aperiam ea ex harum voluptatibus blanditiis architecto expedita minus, molestiae earum enim modi dolore porro! Deserunt, saepe rem.</p>    
+        <a href="#">Link 6</a>
+    </article>
+</div>
+    <h3>Wanna take the pill? Fill the below form and we'll meet soon</h3>
+    <form>
```

Comment:
https://medium.com/nextux/form-design-best-practices-9525c321d759


https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form
Please read above resources carefully, and do the changes if needed.